coderd/database/queries.sql.go

```diff
@@ -3764,27 +3766,29 @@ VALUES (
 	$4,
 	$5,
 	$6,
-	$7
+	$7,
+	$8
 ) ON CONFLICT("name", LOWER(COALESCE(tags ->> 'owner'::text, ''::text))) DO UPDATE SET
 	provisioners = $3,
 	tags = $4,
 	last_seen_at = $5,
 	"version" = $6,
-	api_version = $7
+	api_version = $8
```

Comment:
missing `organization_id` --- is this intentional?

Reply:
I didn't think provisioners could change orgs, but it is actually possible since their share the same auth. I'll add it.